### ISSUE TITLE
[PATCH] Typo in Term::ANSIColor

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#5807]
Migrated from [rt.perl.org#15950](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=15950) (status was 'resolved')

Searchable as RT15950$

**Comment by p5pRT** [comment]
### From mrnobo1024@yahoo.com

This patch fixes a typo in the Term&#8203;::ANSIColor module\.

\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
Do You Yahoo\!?
Yahoo\! Health \- Feel better\, live better
http&#8203;://health\.yahoo\.com

**Comment by p5pRT** [comment]
### From mrnobo1024@yahoo.com

<details><summary>Inline Patch</summary>

```diff
--- lib/Term/ANSIColor.bak	Sun Jul 14 15:55:18 2002
+++ lib/Term/ANSIColor.pm	Fri Aug  2 14:01:52 2002
@@ -227,7 +227,7 @@
 
 This module has two interfaces, one through color() and colored() and the
 other through constants.  It also offers the utility function uncolor(),
-which has to be explicitly imported to be used (see L<SYNOPSYS>).
+which has to be explicitly imported to be used (see L<SYNOPSIS>).
 
 color() takes any number of strings as arguments and considers them to be
 space-separated lists of attributes.  It then forms and returns the escape
```
</details>

**Comment by p5pRT** [comment]
### From @tux

\[mrnobo1024 \- Fri Aug  2 14&#8203;:13&#8203;:54 2002\]&#8203;:




> This patch fixes a typo in the Term&#8203;::ANSIColor module\.




Thanks\, applied as change \#18272 in the development version of Perl

**Comment by p5pRT** [comment]
@tux - Status changed from 'new' to 'resolved'
